src/hub.html

```diff
@@ -359,7 +361,7 @@
                 <a-entity
                     class="pen interactable"
                     body-helper="type: dynamic; mass: 0.001; collisionFilterGroup: 1; collisionFilterMask: 15; gravity: 0 0 0;"
-                    media-loader="src: https://asset-bundles-prod.reticulum.io/interactables/DrawingPen/DrawingPen-34fb4aee27.gltf;"
+                    gltf-model-plus="src: #drawing-pen;"
```

Comment:
interesting, doesn't this change the behavior a bunch? (no loading sound, cube, spawn animation, etc?) probably for the better if it feels good.

Reply:
Yeah, that was my thinking as well, it does feel snappier without the loading and the unnecessary URL resolution.